regen-network/regen-web
# fix: safari horizontal scroll with content overflow APP-1

## Description

Closes: regen-network/rnd-dev-team#1897

This also makes the overflowing sliders scrollable (Getting started section or Project/class details section)



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



On Safari, from https://deploy-preview-2328--regen-marketplace.netlify.app/, go to any page with the section "Resources for Getting started", ie home, /projects, project pages and check that when you scroll to the right with your trackpad, only the slider elements move, not the whole page.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
